SU-HSDO/stanford_mrc
# MRCD8-242 packaged up main menu styles and fine tuned, rm examples

# READY FOR REVIEW

# Summary
- Packaged up Kerri's main menu styles
- Fine-tuned styles
- Removed CSS from remove_me file
- Removed /examples folder since it is no longer being used

# Needed By (Date)
- End of sprint (3/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the main menu in still working (plus has been fine-tuned)

# Affected Projects or Products
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-242

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
